proto/cron/params.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package neutron.cron;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/neutron-org/neutron/x/cron/types";
+
+// Params defines the parameters for the module.
+message Params {
+  option (gogoproto.goproto_stringer) = false;
+  // Admin address that can add/remove schedules
+  string admin_address = 1;
```

Comment:
wait! why do you  need to keep this as part of the cron module params? can't the admin address retrieved from the admin module state?